ckanext/switzerland/templates/package/resource_read.html

```diff
@@ -33,3 +33,59 @@ <h3>{{ _('From the dataset abstract') }}</h3>
   <br>
   {% snippet "package/snippets/terms_of_use.html", pkg=package %}
 {% endblock %}
+
+{% block resource_actions %}
+<ul class="d-flex flex-wrap gap-1 justify-content-end">
+  {% block resource_actions_inner %}
+  {% block action_manage %}
+  {% if h.check_access('package_update', {'id':pkg.id }) %}
+  <li>{% link_for _('Edit resource'), named_route=pkg.type ~ '_resource.edit', id=pkg.name, resource_id=res.id, class_='btn btn-default', icon='pencil' %}</li>
+  {% block action_manage_inner %}{% endblock %}
+  <li>{% link_for _('Views'), named_route=pkg.type ~ '_resource.views', id=pkg.name, resource_id=res.id, class_='btn btn-default', icon='chart-bar' %}
+    {% endif %}
+    {% endblock action_manage %}
+    {% if res.url and h.is_url(res.url) %}
+  <li>
+    <div class="btn-group">
+      <a class="btn btn-primary resource-url-analytics" download="{{ res.name }}" href="{{ res.url }}">
```

Comment:
I copied the whole block from ckan-templates to override it here.
The only thing I changed is adding `download="{{ res.name }}"` to this line. You can also see this by going through the commits of this PR.